apiml-common/src/main/java/org/zowe/apiml/product/eureka/client/ApimlPeerEurekaNode.java

```diff
@@ -428,18 +460,20 @@ public ProcessingResult process(List<ReplicationTask> tasks) {
                         return ProcessingResult.PermanentError;
                     }
                 } else {
+                    networkIssueCounter.success();
                     handleBatchResponse(tasks, response.getEntity().getResponseList());
                 }
             } catch (Throwable e) {
+                networkIssueCounter.fail(e.getLocalizedMessage());
                 if (maybeReadTimeOut(e)) {
                     log.error("It seems to be a socket read timeout exception, it will retry later. if it continues to happen and some eureka node occupied all the cpu time, you should set property 'eureka.server.peer-node-read-timeout-ms' to a bigger value", e);
                     //read timeout exception is more Congestion than TransientError, return Congestion for longer delay
                     return ProcessingResult.Congestion;
-                } else if (isNetworkConnectException(e)) {
-                    logNetworkErrorSample(null, e);
+                } else if (isNetworkConnectException(e) && !networkIssueCounter.isPermanent()) {
+                    logNetworkErrorSample(null, e, "; retrying after delay");
```

Comment:
What is the purpose of this message? Probably it will be more helpful to notify, that the message will be hidden for 10secs...